### ISSUE TITLE
Try Cloakist to change background colour of Airtables

### ISSUE THREAD

**Issue by bryceerobertson** [StampyAI/AISafety.com#114]
![Image](https://github.com/StampyAI/AISafety.com/assets/98026499/7148511a-8640-4f2b-8929-ce582b274fe9)

Currently waiting for Cloakist support to get back to me. Sent follow-up.

**Comment by melissasamworth** [comment]
This would be sweet if you want to do it!

**Comment by triangle42** [comment]
Can I start this task?

**Comment by bryceerobertson** [comment]
Yes, feel free. Although do note that even if it works, it would be a temporary fix since we're going to redesign this page soon

**Comment by triangle42** [comment]
Then I'm in favor for not doing this.
